### DISCUSSION TITLE
Permissions - full list

### DISCUSSION THREAD

**Discussion by KSemenenko** [CommunityToolkit/Maui#1644] [General]
What do you think about adding a list of all the necessary permissions? 
It is not very convenient to write your own custom resolutions and transfer them from project to project.

**Comment by bijington** [comment]
What permissions are you referring to? The ones that the toolkit requires for specific features? Dont we include this in the documentation site?

**Reply by KSemenenko** [reply]
I'm thinking of all the possible permissions. Everything that is available in iOS and Android.

**Reply by bijington** [reply]
Is this to cover a gap from what the MAUI docs doesn't show or something else? Sorry I'm not sure I follow what this will do/be. Do you have an example how it will look?

**Comment by CliffAgius** [comment]
Closed as answered, if closed in error please let us know to re-open.
